datasource/http/test-fixtures/invalid_method.pkr.hcl

```diff
@@ -0,0 +1,26 @@
+source "null" "example" {
+  communicator = "none"
+}
+
+data "http" "basic" {
+  url = "https://www.packer.io/"
+  method = "DELETE"
```

Comment:
If we update the list of supported methods we can change this, otherwise it'll be accepted.